### ISSUE TITLE
Could ssd implements on windows

### ISSUE THREAD

**Issue by duanLH** [weiliu89/caffe#219]
There is a ssd_detection.cpp in the files, I want to know if the SSD can implement on vs? 
thanks

**Comment by weiliu89** [comment]
Yes. It should be. There were several issues discussing about this. There might be some boost issues during compiling. One simple solution is to delete all the codes that use boost to save results in detection_output_layer.c*

**Comment by dtmoodie** [comment]
I have one that is working for detection at github.com/dtmoodie/caffe/tree/merge
Training is somewhat working but I'm running into the issue found here: https://github.com/weiliu89/caffe/issues/198
